api-reference/_hidden/dxFileManagerContextMenu/items/items.md

```diff
@@ -8,11 +8,56 @@ inheritsType: dxFileManagerContextMenuItem
 ---
 ##### shortDescription
 Configures context menu items' settings.
-
+ 
 ---
 
 The **FileManager** widget allows you to add default and create custom context menu items.
```

Comment:
"to add default" add default what?

Reply:
It does not sound right. Why do you need to mention 'default'? It this all the file manager does? The short description above is "configures context menu items' settings", but here you talk about adding and creating items?

Reply:
The context menu can contain default (or predefined) and custom items. Maybe it is not ok to say 'add' about default items and 'create' about custom items. Let's say:

> The **FileManager** widget allows you to add predefined and custom context menu items.

Reply:
Ok